service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -607,6 +609,36 @@ void shouldNotifyCtscAdminOfCMOReadyForJudgeReviewWhenCtscIsEnabled() throws Exc
                 "12345");
         }
 
+        @Test
+        void shouldNotifyRepresentativesOfCMOReadyForPartyReview() throws Exception {
+            CallbackRequest callbackRequest = callbackRequest();
+            CaseDetails caseDetails = callbackRequest().getCaseDetails();
+            CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
+
+            given(representativeService.getRepresentativesByServedPreference(caseData.getRepresentatives(),
+                DIGITAL_SERVICE))
+                .willReturn(getExpectedDigitalRepresentativesForAddingPartiesToCase());
+
+            given(representativeService.getRepresentativesByServedPreference(caseData.getRepresentatives(),
+                EMAIL))
+                .willReturn(getExpectedEmailRepresentativesForAddingPartiesToCase());
+
+            notificationHandler.sendEmailForCaseManagementOrderReadyForPartyReview(
+                new CaseManagementOrderReadyForPartyReviewEvent(callbackRequest, requestData, documentContents));
+            
+            verify(notificationService).sendEmail(
+                eq(CMO_READY_FOR_PARTY_REVIEW_NOTIFICATION_TEMPLATE),
+                eq("fred@flinstone.com"),
+                dataCaptor.capture(),
```

Comment:
These parameters are tested in controller test - is there any benefit to testing them again here? Or is using this OK? (verified that the email has been sent to right address + with right template)

Reply:
Cool, that makes sense - I will add tests in for these parameters